ui/apps/platform/src/Containers/AccessControl/PermissionSets/ResourceDescription.tsx

```diff
@@ -5,6 +5,9 @@ import { ResourceName } from 'types/roleResources';
 // Description for permission resource types. 'Read:' and 'Write:' tokens have
 // special meaning and mark parts related to respective operations.
 const resourceDescriptions: Record<ResourceName, string> = {
+    Access: 'Read: View configuration for authentication and authorization, such as authentication services, roles, groups and users. Write: Modify configuration for authentication and authorization',
```

Comment:
- `.` at the end : )
- be consistent with serial comma use: `... groups, and users...`